Keas.Mvc/Controllers/SystemController.cs

```diff
@@ -36,7 +36,7 @@ public async Task<IActionResult> Emulate(EmulateUserViewModel model)
         {
             //Log.Information("Emulation attempted for {username}", username);
 
-            var user = await _context.Users.SingleOrDefaultAsync(u => u.Email == model.UserEmail || u.Id == model.UserEmail);
+            var user = await _context.Users.SingleOrDefaultAsync(u => u.Email.Equals(model.UserEmail.Trim(), StringComparison.OrdinalIgnoreCase) || u.Id.Equals(model.UserEmail.Trim(), StringComparison.OrdinalIgnoreCase));
```

Comment:
@jSylvestre Maybe do the trim ahead of time and then pass in the trimmed value, instead of trimming twice inside the query.  I'm concerned EF won't generate the query properly with the trim inside (performance wise), but plus it'll be cleaner to just trim once and drop it in a var, since it's a server-side thing